src/appleseed/foundation/math/scalar.h

```diff
@@ -224,6 +224,11 @@ T mix(const T a, const T b, const U x);
 template <typename T, typename U>
 T lerp(const T a, const T b, const U x);
 
+// lerp2() returns the relative position of c between a and b. 
+// it is transformation of lerp(). i.e (c - a) / (b - a)
+template <typename T, typename U>
+T lerp2(const T a, const T b, const U c);
```

Comment:
Rename `c` to `x` for symmetry with `lerp()` since it looks like they're both interpolation parameters?

I wonder if `inverse_lerp()` would make more sense than `lerp2()`?

Reply:
I think `inverse_lerp()` is good too!